ChiModules/LinearBoltzmannSolver/IterativeMethods/lbs_GMRES.cc

```diff
@@ -204,7 +199,8 @@ bool LinearBoltzmann::Solver::GMRES(LBSGroupset& groupset,
         << chunk_overhead_ratio;
       chi_log.Log(LOG_0)
         << "        Sweep Time/Unknown (ns):       "
-        << sweep_time*1.0e9*chi_mpi.process_count/num_unknowns;
+        << sweep_time*1.0e9*chi_mpi.process_count/
```

Comment:
Just a note that at some point, we really need to convert to chrono and duration_cast to avoid things like a random multiply by 1e9.